### PR TITLE
[OpenCL] Cleans debug ops

### PR DESCRIPTION
**Pull request by lukeiwanski**

### CODE REVIEW

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

File: tensorflow/core/kernels/debug_ops.cc [hunk 1]
```diff
@@ -28,25 +28,25 @@ REGISTER_KERNEL_BUILDER(Name("Copy").Device(DEVICE_CPU), CopyOp);
...
+REGISTER_KERNEL_BUILDER(Name("Copy").Device(DEVICE_GPU), CopyOp);
 
 REGISTER_KERNEL_BUILDER(Name("CopyHost")
-                            .Device(DEVICE_SYCL)
+                            .Device(DEVICE_GPU)
                             .HostMemory("input")
                             .HostMemory("output"),
                         CopyOp);
-#endif // TENSORFLOW_USE_SYCL
+#endif
```
Comment (caisq) [inline tensorflow/core/kernels/debug_ops.cc]: Thanks for the clean-ups. Please add "  // GOOGLE_CUDA" at the end of this line.

**Review by caisq — CHANGES_REQUESTED** [review]

File: tensorflow/core/kernels/debug_ops.cc (hunk 1 above)
Comment (lukeiwanski) [inline tensorflow/core/kernels/debug_ops.cc]: It's not really OpenCL related but done in af54bc7 :)

File: tensorflow/core/kernels/debug_ops.cc [hunk 2]
```diff
@@ -36,7 +36,7 @@ REGISTER_KERNEL_BUILDER(Name("CopyHost")
                             .HostMemory("input")
                             .HostMemory("output"),
                         CopyOp);
-#endif
+#endif // GOOGLE_CUDA
```
Comment (caisq) [inline tensorflow/core/kernels/debug_ops.cc]: nit: there should be two spaces before "//"

**Review by caisq — CHANGES_REQUESTED** [review]

File: tensorflow/core/kernels/debug_ops.cc (hunk 2 above)
Comment (lukeiwanski) [inline tensorflow/core/kernels/debug_ops.cc]: Done 0973923

**Review by caisq — APPROVED** [review]

**Comment by lukeiwanski** [comment]
@caisq One of the Linux GPU tests TIMEOUT'ed .. I can't see how that's related to this changeset.

**Comment by caisq** [comment]
Yeah, that timeout was unrelated. Thanks for the PR.

**Comment by lukeiwanski** [comment]
No problem! I have plenty more :D

**Pull request merged**